src/Polly.Core/ResilienceContext.cs

```diff
@@ -0,0 +1,102 @@
+using System.Diagnostics.CodeAnalysis;
+
+namespace Polly;
+
+/// <summary>
+/// An unique assigned to a single execution of <see cref="IResilienceStrategy"/>. It is created manually or automatically
```

Comment:
I don't follow what _"An unique assigned to a"_ is referring to. Seems like it's maybe missing a word?

Reply:
Added "context" there